submissions/Liddell777/hooli-style-popup/index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CatPopup</title>
+    <link rel="stylesheet" href="./CSS/style.css">
+    <link rel="shortcut icon" href="/img/kat_hMS_icon.ico" type="image/x-icon">
+</head>
+<body>     
+        <header>
+            <nav class="nav">
+                <ul class="nav__menu">
+                    <li class="menu__item">
+                        <a href="#" class="menu__link-txt"><img src="img/menu/mail.png" alt="mail"></a>
+                    </li>
+                    <li class="menu__item">
+                        <a href="#" class="menu__item-label"><img src="img/menu/picture.png" alt="Images"></a>
+                    </li>
+                    <li class="menu__item">
+                        <div class="menu">
+                            <input type="checkbox" id="menu-btn" class="menu__input"/>
+                            <label for="menu-btn" class="menu__label" tabindex="0"><img src="img/menu/apps.png" alt="menu"/></label>
+                            <div class="mainpopup mainpopup--invisible">
```

Comment:
Use main-popup instead mainpopop, because it's hard to read

Reply:
Hi, thanks for u comment! I replaced "mainpopop" on "main-popup"